### ISSUE TITLE
Translation: 2 different sentences needed

### ISSUE THREAD

**Issue by ldmpub** [nextcloud/android#1600]
### Actual behaviour
Same key / label is used in 2 different contexts.

### Expected behaviour
To make the app more easy to use:
- need 1 translation in contextual menu => label for an action with a verb like other entries of this menu
- need 1 translation in file properties => label meaning a status (on/off)

See the 2 contexts in screen captures attached with currently the same label in french (confirmed: only one label available in transifex => "**Disponible hors-ligne**"

![label for action](https://user-images.githubusercontent.com/1452146/30933890-27b81610-a3cc-11e7-81b2-0de0c2d8a1e1.png)
![label for status](https://user-images.githubusercontent.com/1452146/30933894-2cbdcea2-a3cc-11e7-8f91-ad64e989e6a0.png)

**Comment by stalker314314** [comment]
This is "set as available offline". I can pick it up. We can even make English translation different as "Set as available offline" (menu) and "Available offline" (in info dialog)?

**Comment by comradekingu** [comment]
@stalker314314 Ensure offline availability, make available offline.
Available on device might be better than available offline

**Comment by ldmpub** [comment]
Thanks for your quick answers and the change already done.

As a translator, when will the new key be available in Transifex for translation? Can I be warned automatically for the arrival of the new key 'favorite_switch' ?

**Comment by tobiasKaminsky** [comment]
It will be available once the PR is merged to master.
If you can get a warning/notification for new strings I do not know :/

**Comment by AndyScherzinger** [comment]
> As a translator, when will the new key be available in Transifex for translation? Can I be warned automatically for the arrival of the new key 'favorite_switch'?

@ldmpub the key will show up tomorrow / I am not aware of any translator notification mechanism for new keys :/

**Comment by ldmpub** [comment]
Translation done some days ago and tested today with success in the app.

Thanks!

**Comment by AndyScherzinger** [comment]
Glad to hear :+1:
